tsqa/plugin/color_plugin.py

```diff
@@ -0,0 +1,918 @@
+"""Color output plugin for the nose testing framework.
+
+Use ``nosetests --with-color`` (no "u"!) to turn it on.
+
+http://en.wikipedia.org/wiki/Rudolph_the_Red-Nosed_Reindeer
+
+"Rudolph the Red-Nosed Reindeer" is a popular Christmas story about Santa
+Claus' ninth and lead reindeer who possesses an unusually red colored nose that
+gives off its own light that is powerful enough to illuminate the team's path
+through inclement weather.
+
+
+Copyright 2007 John J. Lee <jjl@pobox.com>
+
+This code is derived from zope.testing version 3.5.0, which carries the
+following copyright and licensing notice:
+
+##############################################################################
+#
+# Copyright (c) 2004-2006 Zope Corporation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
```

Comment:
i think all we have to do, is add this notice to the NOTICE file, like we do in ats itself
https://github.com/apache/trafficserver/blob/master/NOTICE